File: app/src/main/java/database/TodoBaseHelper.java
package database;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import database.TodoDbSchema.TodoTable;

public class TodoBaseHelper extends SQLiteOpenHelper {

    private static final int VERSION = 1;
    private static final String DATABASE_NAME = "todo.db";

    public TodoBaseHelper( Context context) {
        super(context, DATABASE_NAME, null, VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL("create table " + TodoTable.NAME + "(" +
                        TodoTable.Cols.UUID + " TEXT, " +
                        TodoTable.Cols.TITLE + " TEXT, " +
                        TodoTable.Cols.DETAIL + " TEXT, " +
                        TodoTable.Cols.DATE + " DATE, " +
                        TodoTable.Cols.IS_COMPLETE + " INTEGER)"
        );
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }
}

File: app/src/main/java/database/TodoBaseHelper.kt
package database

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper
import database.TodoDbSchema.TodoTable
import database.TodoDbSchema.TodoTable.Cols

class TodoBaseHelper(context: Context?) : SQLiteOpenHelper(context, DATABASE_NAME, null, VERSION) {
    override fun onCreate(db: SQLiteDatabase) {
        db.execSQL("create table " + TodoTable.NAME + "(" +
                Cols.UUID + " TEXT, " +
                Cols.TITLE + " TEXT, " +
                Cols.DETAIL + " TEXT, " +
                Cols.DATE + " DATE, " +
                Cols.IS_COMPLETE + " BOOLEAN)"
        )
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {}

    companion object {
        private const val VERSION = 1
        private const val DATABASE_NAME = "todo.db"
    }
}